harness/tests/tensorboard/test_base.py

```diff
@@ -74,22 +84,33 @@ def test_unknown_type() -> None:
     with pytest.raises(TypeError, match="Unknown storage type: unknown"):
         env = test_util.get_dummy_env()
         tensorboard.build(
-            env.det_cluster_id, env.det_experiment_id, env.det_trial_id, checkpoint_config
+            env.det_cluster_id,
+            env.det_experiment_id,
+            env.det_trial_id,
+            checkpoint_config,
```

Comment:
I changed those, there were some inconsistency in the file wrt when to put them all in one line versus breaking them up and I was attempting to standardize them. But since this diff is not to reformat, I will revert these changes to avoid confounding the core things we are working on here.